allure-report-face/src/components/widgets-grid/WidgetsGridView.js

```diff
@@ -7,7 +7,9 @@ import {className} from '../../decorators';
 import allurePlugins from '../../pluginApi';
 
 const widgetTpl = (id) => `<div class="widget island" data-id="${id}">
-    <div class="widget__handle fa fa-arrows"></div>
+    <div class="widget__handle">
+        <span class="fa fa-ellipsis-v"></span> <span class="fa fa-ellipsis-v"></span>
```

Comment:
hmm.. for me google shows mostly dots and hand symbol and only a few crossed arrows...
use dots as bg for such long side is bad idea.

Returning back to real problems - we discuss nothing, while this pr can already be solution for lot of people. So i'll do nothing on this pr more. 

p.s. @tk8817 don't notice, @just-boris do it with great love :)